src/rules/prefer-destructured-store-props.ts

```diff
@@ -0,0 +1,130 @@
+import type * as ESTree from "estree"
+import type { TSESTree } from "@typescript-eslint/types"
+import type { AST } from "svelte-eslint-parser"
+import { createRule } from "../utils"
+import { getStringIfConstant } from "../utils/ast-utils"
+
+type NodeRecord = { property: string; node: TSESTree.MemberExpression }
+
+export default createRule("prefer-destructured-store-props", {
+  meta: {
+    docs: {
+      description:
+        "Destructure values from object stores for better change tracking & fewer redraws",
```

Comment:
The first letter should be lowercase to be consistent with existing rules. (see #218)